src/event_pusher/mod_event_pusher_rabbit.erl

```diff
@@ -88,102 +93,65 @@ push_event(Acc, _, _) ->
 %%% Internal functions
 %%%===================================================================
 
+-spec initialize_metrics(Host :: jid:server()) -> ok.
```

Comment:
The spec is incorrect. It returns the list of results of `mongoose_metrics:ensure_metric/3`